ui/src/adapters/api/index.ts

```diff
@@ -23,6 +23,7 @@ export type APIResponse<D> = APISuccessfulResponse<D> | APIErrorResponse;
 
 export enum HTTPStatusError {
```

Comment:
Should this not just be a type union, for consistency?

`export type HTTPStatusError = 0 | 400 | 401 | 404;`